modules/src/core/ics24_host/identifier.rs

```diff
@@ -337,7 +337,7 @@ impl Default for PortId {
 }
 
 #[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
-pub struct ChannelId(String);
+pub struct ChannelId(u64);
```

Comment:
Let's derive `Copy` here? :)

Reply:
PSA: If you do the change, note that `cargo clippy --fix --allow-dirty` can remove all the unnecessary clones for you :)

Reply:
> PSA: If you do the change, note that cargo clippy --fix --allow-dirty can remove all the unnecessary clones for you :)

This is awesome! Thank you! 🙏 Here's the diff -> https://github.com/informalsystems/ibc-rs/pull/2071/commits/b83fa5bcefa533a13c6f2c577fd4a6a3376a2759. Saved me a lot of time! 🎉